src/status_im/contexts/settings/wallet/events_test.cljs

```diff
@@ -129,3 +129,16 @@
                 (sut/make-seed-phrase-keypair-fully-operable
                  cofx
                  [mnemonic-masked password-masked on-success on-error])))))
+
+(deftest make-partly-operable-accounts-fully-operable-test
+  (let [cofx            {:db {}}
+        password-masked (security/mask-data "password")
+        expected        {:fx [[:json-rpc/call
+                               [{:method     "accounts_makePartiallyOperableAccoutsFullyOperable"
```

Comment:
is the typo intentional in the api name? 😅 
accouts -> accounts 🤔

Reply:
Yes 😓